src/v2/view-builder/views/consent/EmailMagicLinkOTPTerminalView.js

```diff
@@ -1,19 +1,23 @@
 import { loc, View } from 'okta';
 import hbs from 'handlebars-inline-precompile';
 
+const generateGeolocationString = (location = {}) => {
+  if (!location.city && !location.country) {
+    return null;
+  }
+  // Use 1 of 2 formats based on presence of 'state' in response:
+  // 1. City, State, Country
+  // 2. City, Country
+  return location.state ?
+    loc('geolocation.formatting.all', 'login', [location.city, location.state, location.country]) :
```

Comment:
Is PM aware that we will display city, state and country in this case?

Reply:
Yup, I've dropped a note to Bhagya, waiting for her response before final merge